src/crimson/mon/MonClient.cc

```diff
@@ -962,18 +997,11 @@ seastar::future<> Client::reopen_session(int rank)
   }
   pending_conns.reserve(mons.size());
   return seastar::parallel_for_each(mons, [this](auto rank) {
-    // TODO: connect to multiple addrs
```

Comment:
please note, this TODO still applies after this change. but since it'd be a more thorough endeavour. it's fine to remove it, i think.